model-api/src/commonMain/kotlin/org/modelix/model/api/INode.kt

```diff
@@ -380,6 +380,20 @@ fun INode.tryResolveProperty(role: String): IProperty? {
         ?: allLinks.find { it.getSimpleName() == role }
         ?: allLinks.find { it.getUID() == role }
 }
+
+/**
+ * Resolves whether the child link is ordered or not.
+ */
+fun INode.isChildRoleOrdered(role: String?): Boolean {
+    // Assume children to be ordered by default.
+    // Unordered children are the special case.
```

Comment:
Maybe that could be lifted to the kdoc. That's information potentially interesting for users of this method.

Reply:
Done. Also I extended the explanation further.